docs/apidocs/index.rst

```diff
@@ -4,6 +4,93 @@
 Qiskit Experiments API Reference
 ================================
 
+To execute useful algorithms with quantum computing systems,
+you need to start from scrutinizing the physical parameters of a quantum processor (:ref:`characterization`)
+and carefully design stimulus pulses tailored for each qubit (:ref:`calibration`)
+to maximize the fidelity of your quantum state operation constituting
+the instruction set architecture, or `basis gates` in the Qiskit terminology.
+Once this process has completed, you need to evaluate different levels of performance metrics
+from a single qubit gate error to the quantum volume for a group of qubits (:ref:`verification`).
+
+During this workflow, experimentalists need to deal with many different experiments and
+need to manage massive amount of analysis data.
+Furthermore, physical properties of qubits may change from time to time,
+and this operation is often repeated on a regular basis while keeping the past data available for the time series analysis.
+This is where the Qiskit Experiments comes into play.
+This project aims at lightening the burden of experimentalists with the circuit generation and
+data analysis framework implemented upon the Qiskit core package with preset experiment libraries.
+
+
+Overview
+========
+
+.. rubric:: Running experiment
+
+A workflow of the experiment can be roughly divided into two operations without loss of generality, namely,
+generation of the experiment program and analysis of data returned from the quantum system.
+These operations are embodied as a python class which we call `experiment class` and `analysis class`.
+
+This illustrates a typical workflow in Qiskit Experiments.
+
+.. code-block:: python
+    :linenos:
+    :emphasize-lines: 3,6,11
+
+    from qiskit_experiments.library import MyExperiment
+
+    exp = MyExperiment(qubits=[0, 1])
+    exp.set_experiment_options(**options)
+
+    exp_data = exp.run(backend)
+
+    # wait for backend queue
+    exp_data.block_for_results()
+
+    exp_data.save()
+
+
+An experiment class contains the default analysis class, thus you just need to import the experiment class
```

Comment:
Details on anlaysis classes aren't needed here, they are mentioned in framework. All you need to start is that running an experiment runs circuits and then performs analysis and returns the results.